news/2021-09-26-project-loved-september-2021.md

```diff
@@ -0,0 +1,404 @@
+
+Although 280 BPM sounds quite intimidating, all these seven mappers have made sure the maps are entertaining and comfortable to play. The ones made by 7_7 and uone are quite well structured, with easy patterns that are draining but not unfair in the slightest, while the others crafted by Shallty and rezi888 are perfect examples of vocal/melody mapping, enabling players to keep up with the patterns with ease even through the chaotic streams.
+
+Still, if you want to know what every one of these maps has to offer, rest assured, none of these maps is any similar to the other, proving that every mapper can make a unique approach to the music. Enjoy!
+
+[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/1334132.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Official HIGE DANdism - Bad for me (Dz'Xa's Amenpunk)](https://osu.ppy.sh/beatmapsets/1334132#taiko) by [KTYN](https://osu.ppy.sh/users/2250574) and [xpokn](https://osu.ppy.sh/users/5115345)\
+200 BPM, 3:08 | 5.73★\
+*description written by [-Kazu-](https://osu.ppy.sh/users/920861)*
+
+In the same light as *[Itte.](https://osu.ppy.sh/beatmapsets/971578#taiko/2034239)* from the July 2021 voting, we've found those breakcore renditions of J-pop / pop-rock songs enjoyable and engaging to play. Therefore, for this round, we brought *Bad for me (Dz'Xa's Amenpunk)* by Official HIGE DANdism!
+
+As expected, KTYN is good at taking these songs and converting them into absolute bangers of a map to ensure it receives the appreciation it deserves. With a map suitable for entry-level Inner Oni players, this is very likely to be one that you'll enjoy playing, as KTYN's style focuses so much on being fun before anything else. Don't miss it!
+
+[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/389617.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[daisan - -+](https://osu.ppy.sh/beatmapsets/389617#taiko) by [aabc271](https://osu.ppy.sh/users/155707), [ll-oscar](https://osu.ppy.sh/users/1488736), [Nardoxyribonucleic](https://osu.ppy.sh/users/876419), [Prophecy](https://osu.ppy.sh/users/3793769) and [Skylish](https://osu.ppy.sh/users/2845958)\
+185 BPM, 2:05 | 1.44★ – 5.77★\
+*description written by [POCARI SWEAT](https://osu.ppy.sh/users/5082685)*
+
+If you played osu!taiko for a long time, you might have heard the name *-+* at least once. Mapped by 5 veteran osu!taiko mappers from the Hong Kong community, *-+*'s osu!taiko diffs have finally made their way to Loved. There are 8 osu!taiko difficulties mapped with star ratings ranging between 1.44\\\~5.77\\\*, containing varying patterns and decent BPM structures, with the highest one serving you a pile of 1/6 patterns, albeit relatively subdued. If you're a professional player tired of playing a hell of intricate maps, or a beginner looking for a quality one to practice, this one will be a great choice.
```

Comment:
I honestly have no idea what happened here ([twice](https://github.com/ppy/osu-wiki/pull/6165/commits/0beeb90fc2c4ec5c271a3687451aa10bfa9fde33#diff-b3c13c079b9d40eb11b289196e228148f79a999151e594c11802829ba8300020L175-R175) [now](https://github.com/ppy/osu-wiki/pull/6165/commits/b99328e99fe0e751416f42fbb389c1daeaf2933e#diff-b3c13c079b9d40eb11b289196e228148f79a999151e594c11802829ba8300020L175-R175)). Did you look through the diff of your own commit? Did some tool try to automatically escape backslashes?

```suggestion
If you played osu!taiko for a long time, you might have heard the name *-+* at least once. Mapped by 5 veteran osu!taiko mappers from the Hong Kong community, *-+*'s osu!taiko diffs have finally made their way to Loved. There are 8 osu!taiko difficulties mapped with star ratings ranging between 1.44\~5.77\*, containing varying patterns and decent BPM structures, with the highest one serving you a pile of 1/6 patterns, albeit relatively subdued. If you're a professional player tired of playing a hell of intricate maps, or a beginner looking for a quality one to practice, this one will be a great choice.
```